src/shared/components/HamburgerButton/HamburgerButton.style.ts

```diff
@@ -14,7 +14,7 @@ export const Hamburger = styled.button`
   top: 18px;
   // FIXME: this shouldn't be set to 'fixed', button should work on its own, consumer may set it to fixed
   position: fixed;
-  z-index: ${zIndex.header + 1};
+  z-index: ${zIndex.sideNav + 1};
```

Comment:
I know that this incrementation was here before, but maybe since we're adding `sideNav` index, we could also do `nearSideNav` with value like 210 or so.